### DISCUSSION TITLE
Hide private repositories from followers' feed

### DISCUSSION THREAD

**Discussion by umang-malik** [community/community#21672] [New to GitHub]
<p>If a user creates a public repository and then changes it to private, the post still shows on his followers’ feed as:</p>
<p><em>"user</em> created a repository <em>user/privateRepo."</em></p>
<p>I personally believe that if a user privates a repository, then such old posts should be hidden from his followers’ feed. Since the user may have been involved in a secret project, but accidently created a public repository with that project name. Discussion on the suggestion is welcome :slightly_smiling_face:</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Thanks for the feedback! I’ve taken your suggestion and passed it along to the appropriate teams. Thanks again for reaching out :grinning:</p>

**Reply by kreee00** [reply]
is it implemented already? so if im involved in a secret project, will my followers see what im doing?
